.github/workflows/main.yml

```
@@ -20,14 +20,26 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            optional_args: "-pt"
+          - os: windows-latest
+            optional_args: "-pt --cmake_gen ninja"
```

```suggestion
            optional_args: -pt --cmake_gen ninja
```

Not sure we need these?